src/content/app/entity-viewer/variant-view/variant-summary-strip/VariantSummaryStrip.module.css

```diff
@@ -24,10 +24,10 @@
 
 .emphasized {
   font-weight: var(--font-weight-bold);
+  margin-right: 12px;
```

Comment:
This contradicts the name of the class. And also, you no longer seem to be using this class anywhere; you are using `featureStripStyles.featureNameEmphasized` in `VariantSummaryStrip` component

Reply:
It's probably the other `VariantSummaryStrip` component that you are talking about. This class is used in `src/content/app/entity-viewer/variant-view/variant-summary-strip/VariantSummaryStrip.tsx`.